server/sonar-server/src/main/java/org/sonar/server/plugins/ws/InstalledPluginsWsAction.java

```diff
@@ -78,21 +93,44 @@ public void handle(Request request, Response response) throws Exception {
       );
   }
 
-  private void writeMetadataList(Response response, Collection<PluginMetadata> pluginMetadatas) {
-    JsonWriter jsonWriter = response.newJsonWriter();
-    jsonWriter.beginArray();
-    for (PluginMetadata pluginMetadata : pluginMetadatas) {
-      writeMetadata(jsonWriter, pluginMetadata);
-    }
-    jsonWriter.endArray();
-    jsonWriter.close();
+  private void writeMetadataList(JsonWriter jsonWriter, Collection<PluginMetadata> pluginMetadatas) {
+      jsonWriter.name(OBJECT_PLUGINS);
+      jsonWriter.beginArray();
+      for (PluginMetadata pluginMetadata : pluginMetadatas) {
+        writeMetadata(jsonWriter, pluginMetadata);
+      }
+      jsonWriter.endArray();
   }
 
   private void writeMetadata(JsonWriter jsonWriter, PluginMetadata pluginMetadata) {
     jsonWriter.beginObject();
+
     jsonWriter.prop(PROPERTY_KEY, pluginMetadata.getKey());
     jsonWriter.prop(PROPERTY_NAME, pluginMetadata.getName());
+    jsonWriter.prop(PROPERTY_DESCRIPTION, pluginMetadata.getDescription());
     jsonWriter.prop(PROPERTY_VERSION, firstNonNull(pluginMetadata.getVersion(), DEFAULT_VERSION));
+    jsonWriter.prop(PROPERTY_LICENSE, pluginMetadata.getLicense());
+
+    writeOrganisation(jsonWriter, pluginMetadata);
+
+    writeUrls(jsonWriter, pluginMetadata);
+
+    jsonWriter.endObject();
+  }
+
+  private void writeOrganisation(JsonWriter jsonWriter, PluginMetadata pluginMetadata) {
+    jsonWriter.name(OBJECT_ORGANIZATION);
+    jsonWriter.beginObject();
+    jsonWriter.prop(PROPERTY_NAME, pluginMetadata.getOrganization());
```

Comment:
I don't know why, but many fields of pluginMetadata (organization, org url, urls, license, ...) do not return null but empty strings when missing. For this reason the JSON is quite strange: 
```
{
  "key": "uiplugin",
  "name": "SonarQube Integration Tests :: Plugins :: UI",
  "description": "Main plugin for UI tests",
  "version": "1.0-SNAPSHOT",
  "license": "",
  "organization": {
    "name": "",
    "url": ""
},
"urls": {
  "homepage": "",
  "issueTracker": ""
}
}
```